frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java

```diff
@@ -32,6 +44,7 @@
 public abstract class HttpRequestHandler extends SimpleChannelInboundHandler<FullHttpRequest> {
 
     private static final Logger logger = LoggerFactory.getLogger(HttpRequestHandler.class);
+    Map<String, ModelServerEndpoint> endpointMap;
```

Comment:
I think this should be protected

Reply:
Any particular reason why this should be protected? I don't foresee anyone outside of this package using this variable. All its subclasses are in the same package, so shouldn't we leave this as "default" instead of "protected"?